DEVELOP.rst

```diff
@@ -10,28 +36,29 @@ directory:
 
 .. code-block:: console
 
-   $ mkdir -pv ${GOPATH}/src/github.com/crate
-   $ cd ${GOPATH}/src/github.com/crate
-   $ git clone https://github.com/crate/cratedb-prometheus-adapter.git
-   $ cd cratedb-prometheus-adapter
+   mkdir -pv ${GOPATH}/src/github.com/crate
+   cd ${GOPATH}/src/github.com/crate
```

Comment:
Is this actually required, @chaudum? On my machine, it just works within the root directory of the repository's working tree.

Reply:
You can check out the repository to any location on your local machine. However, it is common, that people organize their go projects in the `$GOPATH/src`, because it is the place where Go modules are located. Please refer to https://golang.org/doc/code for more information.